OpenSim/Common/Exception.h

```diff
@@ -50,36 +50,50 @@
 namespace OpenSim {
 
 
-/** Macros to throw OpenSim exceptions. The purpose of these macros is to avoid
-having to provide the first few arguments (which are common to all OpenSim
-exceptions) to the exception constructor in the throw statements. This also
-allows us to add more details (eg class name) later easily.                   */
+/** @name Macros to throw OpenSim exceptions
+The purpose of these macros is to avoid having to provide the first few
+arguments (which are common to all OpenSim exceptions) to the exception
+constructor in the throw statements.
```

Comment:
I think the purposes, in order of decreasing importance are:
- ensure consistency of message formatting
- ensure that all messages contain file/line/func information
- make it easy for developers to produce good messages
- save typing